freqtrade/edge/__init__.py

```diff
@@ -168,20 +192,21 @@ def adjust(self, pairs) -> list:
         """
         Filters out and sorts "pairs" according to Edge calculated pairs
         """
-
         final = []
         for pair, info in self._cached_pairs.items():
             if info.expectancy > float(self.edge_config.get('minimum_expectancy', 0.2)) and \
                 info.winrate > float(self.edge_config.get('minimum_winrate', 0.60)) and \
                     pair in pairs:
                 final.append(pair)
 
-        if final:
-            logger.info('Edge validated only %s', final)
-        else:
-            logger.info('Edge removed all pairs as no pair with minimum expectancy was found !')
+        if self._final != final:
```

Comment:
any chance to name this `_final_pairs` or similar? "final" can be anything or nothing ...

Reply:
will do !